### ISSUE TITLE
wxWidgets debug alert: failing to access a file/folder stops Toucan

### ISSUE THREAD

**Issue by cricri-pingouin** [PortableApps/Toucan#39]
I get a message "assert IsOpened() failed in GetFirst()" as in the attached screenshot.
If you choose "No" or "Cancel", Toucan will continue the backup. Annoying if you a large backup, leave it unattended, and later find out it stopped and waiting for you to click a button.

![Toucan error](https://user-images.githubusercontent.com/64480118/102710349-957f8280-42a9-11eb-9ff1-4514b52b3694.PNG)

**Comment by cricri-pingouin** [comment]
This keeps on interrupting my backups, and it does it with files that are not in use. Not sure if those files have some special attributes that cause the error.
On inspecting one folder casuing the problem:
- Long path, not sure if there is a limit in full path length?
- Some symbols @ in the files/folders names
- Some symbols _ in extensions

In the meantime, I will probably revert to v3.0.4 again.

**Comment by alex-ter** [comment]
Yeah, that's kind of a feature (the wxWidgets are compiled in debug mode to make it easier to troubleshoot things), but I see your point. Probably it's time to ditch the debug build, as it hasn't been much of use anyway.

**Comment by cricri-pingouin** [comment]
This is the actual error from the log:
`18:34:27	Failed to open 'G:\Utils\Files\Some Program\Data\Some Program\roaming\modified\@APPDATA@' for writing (error 0: the operation completed successfully.)
18:34:27	Failed to modify file times for 'G:\Utils\Files\Some Program\Data\Some Program\roaming\modified\@APPDATA@\' (error 0: the operation completed successfully.)`
